[main] Upgrade to latest dependencies

knative-automation opened the pull request:
Cron -knative-prow-robot

/cc knative-sandbox/channel-wg-leads
/assign knative-sandbox/channel-wg-leads

Produced by: knative-sandbox/knobots/actions/update-deps

Details:
```
bumping knative.dev/eventing d2431c7...e3dbc74:
  > e3dbc74 upgrade to latest dependencies (# 5393)
  > af770af Update actions (# 5392)
  > 1bbc013 [rekt] Adding smoke test for parallel (# 5390)
  > c5850a1 [rekt ]adding sequence smoke test (# 5391)
  > 6afd14f adding a method to collect retry event count (# 5115)
  > 8e157ad scale back the documentation used for ContainerSource spec.template. (# 5385)
  > 8ec2987 Call runCleanup() after collecting logs (# 5387)
  > d64e7f4 Collect logs for knative-eventing-logs only when test failed (# 5386)
bumping knative.dev/networking 7267fe5...ace2d33:
  > ace2d33 upgrade to latest dependencies (# 408)
bumping knative.dev/pkg 620e8d2...4564797:
bumping knative.dev/reconciler-test ed3c378...d5adf72:
  > d5adf72 upgrade to latest dependencies (# 200)
```

benmoss commented:
Syntax error has been fixed but not in time: https://github.com/knative-sandbox/.github/pull/110

lionelvillard commented:
/test pull-knative-sandbox-eventing-kafka-integration-test-mt-source

knative-automation committed:
upgrade to latest dependencies

bumping knative.dev/eventing d2431c7...e3dbc74:
  > e3dbc74 upgrade to latest dependencies (# 5393)
  > af770af Update actions (# 5392)
  > 1bbc013 [rekt] Adding smoke test for parallel (# 5390)
  > c5850a1 [rekt ]adding sequence smoke test (# 5391)
  > 6afd14f adding a method to collect retry event count (# 5115)
  > 8e157ad scale back the documentation used for ContainerSource spec.template. (# 5385)
  > 8ec2987 Call runCleanup() after collecting logs (# 5387)
  > d64e7f4 Collect logs for knative-eventing-logs only when test failed (# 5386)
bumping knative.dev/networking 7267fe5...ace2d33:
  > ace2d33 upgrade to latest dependencies (# 408)
bumping knative.dev/pkg 620e8d2...4564797:
bumping knative.dev/reconciler-test ed3c378...d5adf72:
  > d5adf72 upgrade to latest dependencies (# 200)

Signed-off-by: Knative Automation <automation@knative.team>

matzew reviewed (approved):
/lgtm
/approve
